queries.sql

```diff
@@ -195,4 +195,98 @@ WHERE
 	date_of_birth BETWEEN '1990-01-01'
 	AND '2000-01-01'
 GROUP BY
-	species;
\ No newline at end of file
+	species;
+
+-- Create owners table
+CREATE TABLE owners(
+	id serial PRIMARY KEY,
+	full_name varchar(255),
+	age INT,
+);
+
+-- Create species table
+CREATE TABLE species(id serial PRIMARY KEY, name varchar(255));
+
+-- Drop/delete species column from animals table
+ALTER TABLE
+	animals DROP COLUMN species;
+
+-- Connect species table into animals table
+ALTER TABLE
+	animals
+ADD
+	species_id INT,
+ADD
+	CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id) ON DElETE CASCADE;
+
+-- Connect owners table into animals table
+ALTER TABLE
+	animals
+ADD
+	owners_id INT,
+ADD
+	CONSTRAINT fk_owners FOREIGN KEY (owners_id) REFERENCES owners(id) ON DElETE CASCADE;
+
+SELECT
+	animals.name
+FROM
+	animals
+	JOIN owners ON animals.owners_id = owners.id
+WHERE
+	owners.full_name = 'Melody Pond';
```

Comment:
- [ ] Great job so far @Nasirkhan294 There is just a single change I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] I noticed that from the screenshots you attached in your PR, this first query isn't returning the right results so kindly fix this as there seems to be a mistake in your query. Your query can look something like this below :+1: 
```ruby
   SELECT * FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   WHERE full_name = 'Melody Pond';
```

| Your Results | Expected Results |
|--------|--------|
| ![image](https://github.com/Nasirkhan294/vet-clinic-DB/assets/102699958/0419ba23-c74f-4535-9da5-41ca4fdd35fd) | ![image](https://github.com/Nasirkhan294/vet-clinic-DB/assets/102699958/6c9f1f99-df6d-460d-be3a-42dc6098716e) |

Reply:
Hello @Nasirkhan294 I mistakenly approved the PR on the dashboard instead is requesting changes but it will be reverted soon so you can go on with the changes and request a second review when it's done :+1:

Reply:
Hi @NahnahAJ, Sure, I will do that.